src/main/java/seedu/address/logic/parser/FindCommandParser.java

```diff
@@ -19,15 +26,42 @@ public class FindCommandParser implements Parser<FindCommand> {
      * @throws ParseException if the user input does not conform the expected format
      */
     public FindCommand parse(String args) throws ParseException {
-        String trimmedArgs = args.trim();
+        requireNonNull(args);
+        ArgumentMultimap argMultimap =
+                ArgumentTokenizer.tokenize(args, PREFIX_NAME, PREFIX_ADDRESS);
+
+        if (arePrefixesPresent(argMultimap, PREFIX_NAME, PREFIX_ADDRESS)
```

Comment:
Kinda awkward way to specify there can only be one prefix. Might be difficult to generalise.
May I suggest modifying :arePrefixesPresent?